lib/pronto/github.rb

```diff
@@ -70,7 +70,7 @@ def create_pull_request_review(comments)
         comments: comments.map do |comment|
           {
             path:     comment.path,
```

Comment:
Layout/HashAlignment: Align the keys of a hash literal if they span more than one line.